ibis/pandas/api.py

```diff
@@ -7,3 +7,24 @@
 
 def connect(dictionary):
     return PandasClient(dictionary)
+
+
+def from_dataframe(df, name='df', client=None):
```

Comment:
Sorry, I don't think I was clear here. What I meant was an *instance* of `PandasClient`, so what you want is something like:

```python
if client is None:
    return connect({name: df}).table(name)
client.dictionary[name] = df
return client.table(name)
```

Reply:
pushed as you suggested. this *does* mutate the client, is that ok?

Reply:
Yeah, I think that's fine. Can you add some documentation to the `client` parameter to this effect?